### ISSUE TITLE
Look into DNS false positives for OpenVPN detector

### ISSUE THREAD

**Issue by keithjjones** [zeek/spicy-analyzers#46]
I'm seeing a few false positives with dns on a busy university network and a commercial company network:

```
$ cat conn.log | python ~/Source/zeek-sanitize/sanitize.py 2,4
1617725133.905938	Cporeo3QsXCzgKXKY1	REDACTED-1	53847	REDACTED-2	53	udp	dns,spicy_openvpn_udp_hmac	-	-	-	S0	-	-	0	D	1	71	0	0	-	F	-	-	-	-	US	US	-	4020	-
1617725239.404398	CocZRy4L3lrODcxRu2	REDACTED-3	35761	REDACTED-2	53	udp	dns,spicy_openvpn_udp_hmac	-	-	-	S0	-	-	0	D	1	72	0	0	-	F	-	-	-	-	US	US	-	4020	-
1617725292.306323	CHMIrI1rocG1RPK2E7	REDACTED-4	60848	REDACTED-5	53	udp	dns,spicy_openvpn_udp_hmac	-	-	-	S0	-	-	0	D	1	71	0	0	-	F	-	-	-	-	-	US	-	4020	-
1617725372.270816	Cj4ncuA8sCnDqHd69	REDACTED-3	43049	REDACTED-2	53	udp	dns,spicy_openvpn_udp_hmac	-	-	-	S0	-	-	0	D	1	72	0	0	-	F	-	-	-	-	US	US	-	4020	-
1617725546.742503	Cqv7YX1R6ym7VLjLHj	REDACTED-6	53871	REDACTED-7	53	udp	dns,spicy_openvpn_udp_hmac	-	-	-	SHR	-	-	0	^d	0	0	1	70	-	F	-	-	-	-	-	US	-	4020	-
1617725546.742437	Cc4w7v1An2rJ4Kc2q3	REDACTED-8	50675	REDACTED-7	53	udp	dns,spicy_openvpn_udp_hmac	-	-	-	SHR	-	-	0	^d	0	0	1	70	-	F	-	-	-	-	US	US	-	4020	-
```

The resp IP addresses are 8.8.8.8 in some cases (but not all).  It seems to only happen on a packet or two sent in one direction.  Very occasionally you will have multiple packets going in one direction, or a single packet going in each direction, but the bulk of the FP detections are a single packet like the log above.  I've looked into the DNS protocol and there isn't anything I see in there I can easily filter out in the Openvpn analyzer.  I'm open to suggestions.  When I looked for openvpn handshakes in the past @0xxon recommended that I didn't.  This is what is detected when I don't.  I'm creating this issue here because I'm out of ideas how to address this and could use a hand.  Thanks!

**Comment by keithjjones** [comment]
Another note: I am purposefully using the original DNS analyzer on these sensors, and not Spicy's, because I'm just testing the OpenVPN and IPSec components at this time, if you think the logs above look weird.  :)

**Comment by keithjjones** [comment]
I'm pretty consistently seeing this false positive when there are few packets, and one side of the connection seems to be dropped.  I'm not sure we can code around that situation.

**Comment by keithjjones** [comment]
I tested a fix today (MR #73) that requires packets in both directions.  It kills off the false positives I was seeing.
